.github/workflows/build-instructions.yml

```diff
@@ -26,3 +26,43 @@ jobs:
           cmake -S . -B build && \
           cmake --build build && \
           cmake --install build'
+
+  pip_install:
+    name: Pip install on  ${{ matrix.config.os }}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - os: windows-2019
+            preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
```

Comment:
Cannot CMake usually take care of sourcing the right compiler and environment?

Reply:
Ah, it might be compounded by us using the Ninja generator - not so much CMake assuming MinGW, but Ninja.

Reply:
Ah, that would make sense.

Reply:
If there is any way we can remove this malarky that would be _amazing_ (maybe wrap this into #538?)

Reply:
I kinda like having Ninja as a cross-platform consistent generator, which as a bonus trivially parallelizes builds. But if it can completely remove the need for vcvarsall in CI, then removing it (or making it conditional on OS) is probably worthwhile.

Reply:
Ninja is nice indeed as it speeds up builds on Windows. Maybe take a look at OCIO (which I think use Ninja too) and see how they do it?

Reply:
> maybe wrap this into #538

That would get my vote, but happy to try it out in this PR if you like.

Reply:
> Maybe take a look at OCIO

They skip Ninja on Windows (https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/660cf515faa491fadab188eb0f2ed62b30b4b4f6/setup.py#L93) because

> MSVC would require all variables be exported for Ninja to pick it up, which is a little tricky to do

:laughing:

Reply:
Lets save this for another day

Reply:
Added an investigation ticket to check feasibility of using the MSVC generator #669